spec/fixtures/invalid-typechecked/tslint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-unused-variable": true
```

Comment:
Related, [TSLint’s tests](https://github.com/palantir/tslint/tree/master/test/rules/no-unused-variable) for `no-unused-variable` use the `tslint.json` below. Should I add `"linterOptions": { "typeCheck": true }` to the `tslint.json` files I added in this PR?

```JSON
{
  "linterOptions": {
    "typeCheck": true
  },
  "rules": {
    "no-unused-variable": true
  }
}
```